src/components/DocumentationTopic.vue

```diff
@@ -755,15 +757,8 @@ export default {
 }
 
 .full-width-container .doc-content .minimized-container {
-  padding-left: 20px;
-  padding-right: 20px;
-
-  @include inTargetIde {
-    @include breakpoint(xsmall) {
```

Comment:
Should we remove the `xsmall` breakpoint entirely or do you anticipate us still needing it? It seems like it was only used in the deleted code from this PR.